Makefile

```diff
@@ -0,0 +1,13 @@
+install:
+	npm install --no-package-lock --no-audit
```

Comment:
No need for the `--no-package-lock` anymore since you now have a package.lock anyway :) 

```suggestion
	npm install --no-audit
```